### ISSUE TITLE
This bot steals your money

### ISSUE THREAD

**Issue by adee000** [PancakeSwapV2Lib/pancakeswap-v2-periphery#1]
see: https://github.com/PancakeSwapV2Lib/pancakeswap-v2-periphery/blob/main/contracts/interfaces/IPancakeswapV2Factory.sol

uniswapDepositAddress() gets called in for the called "trading bot" but this is just used to send money to hardcoded address

**Comment by Solly88** [comment]
agree. lets report. linked to this https://www.youtube.com/watch?v=weKtGkDLFI4&t=183s
